tests/test_extra_jinja2_keys_migrator.py

```diff
@@ -8,7 +8,7 @@
 
 from test_migrators import run_test_migration
 
-VERSION_CF = Version(piggy_back_migrations=[ExtraJinja2KeysCleanup()])
+VERSION_CF = Version(piggy_back_migrations=[ExtraJinja2KeysCleanup()], check_solvability=False)
```

Comment:
Can we use a default keyword to avoid having to put this everywhere?

Reply:
The default is true. It's turned to false because these tests don't use real feedstocks

Reply:
Ohhhh. I had assumed we would set the default to False to maintain backwards compat and then set it to true where we wanted. I'm happy either way though.

Reply:
I thought we'd want to use it in more cases than not. It's set to false be cause most tests aren't backed by a real package. I can flip it if you want.

Reply:
nope it is fine as is.